src/app/content/components/Page/MediaModal.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import renderer, { act } from 'react-test-renderer';
+import MediaModal from './MediaModal';
+import React from 'react';
+
+describe('MediaModal', () => {
+  const childContent = <div>Test Content</div>;
```

Comment:
You could define the entire Component to be rendered so that each test just calls
`renderer.create(<Component />)` or `renderer.create(<Component isOpen={false} />)`